src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,12 @@
+export class UpdateUserDto {
+    id: string;
+    first_name?: string;
+    last_name?: string;
+    password?: string;
+    email?: string;
+    bio?: string;
+    created_at: Date;
```

Comment:
since this dude is an update model, `created_at` might not be updated here, instead `updated_at` would suit it more